media/js/firefox/new/scene2.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// set up namespace
+if (typeof Mozilla == 'undefined') {
+    var Mozilla = {};
+}
+
+Mozilla.Firefox = window.Mozilla.Firefox || {};
+
+;(function($, dataLayer, MFNew) {
+    'use strict';
+
+    var $directDownloadLink = $('#direct-download-link');
+    var $stage = $('#stage');
+    var downloadStarted = parseInt(MFNew.params.get('dlstarted'), 10) === 1;
+
+    // pixel remains in perpetuity (https://bugzilla.mozilla.org/show_bug.cgi?id=1222945#c2)
+    if (!window._dntEnabled()){
```

Comment:
As this is probably the least important bit of code on the page, is it worth moving it further down to just before the window `'load` conditional? Could probably be contained/called as a function seeing as it creates some variables that nothing else needs reference to?